odl/operator/operator.py

```diff
@@ -709,33 +937,21 @@ def _apply(self, x, out):
         >>> op = IdentityOperator(r3)
         >>> x = r3.element([1, 2, 3])
         >>> out = r3.element()
-        >>> OperatorSum(op, op)(x, out)
+        >>> OperatorSum(op, op)(x, out)  # In place, returns out
         Rn(3).element([2.0, 4.0, 6.0])
         >>> out
         Rn(3).element([2.0, 4.0, 6.0])
-        """
-        # pylint: disable=protected-access
-        tmp = (self._tmp_ran if self._tmp_ran is not None
-               else self.range.element())
-        self._op1._apply(x, out)
-        self._op2._apply(x, tmp)
-        out += tmp
-
-    def _call(self, x):
-        """Return ``op(x)``.
-
-        Examples
-        --------
-        >>> from odl import Rn, ScalingOperator
-        >>> r3 = Rn(3)
-        >>> A = ScalingOperator(r3, 3.0)
-        >>> B = ScalingOperator(r3, -1.0)
-        >>> C = OperatorSum(A, B)
-        >>> C(r3.element([1, 2, 3]))
+        >>> OperatorSum(op, op)(x)  # In place, returns out
```

Comment:
~~That the in-place method returns out, thus explaining the printout on the next line.~~ Oh, this is the out-of-place one. Then the comment has to go.